README.md

```diff
@@ -511,14 +510,14 @@ enabled=true
 enabled=true
 ```
 
-NOTE: The above configuration is the minimum configuration needed to enable tracing and metrics. With these configurations default values are load as the other configuration parameters of metrics and tracing.
+NOTE: The configurations listed above are the minimum configurations needed to enable tracing and metrics. These load the default values of metrics and tracing.
```

Comment:
@kasun04 : "These load the default values of metrics and tracing." Does this give out the correct meaning?